### DISCUSSION TITLE
How to add a logo on badge-maker?

### DISCUSSION THREAD

**Discussion by alexandresanlim** [badges/shields#8393] [Q&A] [answered]
I'm using the [badge-maker package](https://github.com/badges/shields/blob/master/badge-maker/README.md) to build a badge on my aplication. Is it possible add a logo? looking at the parameters it seems not

![image](https://user-images.githubusercontent.com/5353685/189493918-37203b1e-f92d-47d7-bcd2-bffc9661599c.png)

**Comment by chris48s** [comment, marked as answer]
It is currently not possible to add a logo.

We've got an open issue for adding logos to badge-maker at https://github.com/badges/shields/issues/4947 but it kinda stalled because we failed to arrive at a consensus around what we're going to do with the custom logos.
